multi/stm32l4-multi/stm32l4-multi.c

```diff
@@ -41,6 +42,30 @@
 #define STACKSZ 640
 
 
+#if (UART1 && TTY1) || (UART2 && TTY2) || (UART3 && TTY3) || \
+	(UART4 && TTY4) || (UART5 && TTY5)
+#error "Can't use UART as UART and TTY at the same time!"
+#endif
+
+#ifndef UART_CONSOLE
+#error "UART_CONSOLE not specified"
+#endif
+
+#if !(UART_CONSOLE == 1 && (UART1 || TTY1)) && \
+	!(UART_CONSOLE == 2 && (UART2 || TTY2)) && \
+	!(UART_CONSOLE == 3 && (UART3 || TTY3)) && \
+	!(UART_CONSOLE == 4 && (UART4 || TTY4)) && \
+	!(UART_CONSOLE == 5 && (UART5 || TTY5))
+#warning "Console enabled on disabled UART/TTY"
```

Comment:
Thinked about it if this should be treated as warning or error
In fact this is currently the only way to fastly silence the console - set it to invalid UART.
Decided to warning, but I'm open to suggestion